Source/Core/DolphinWX/InputConfigDiag.h

```diff
@@ -83,6 +84,37 @@ class PadSettingCheckBox : public PadSetting
 	ControllerEmu::ControlGroup::Setting* const setting;
 };
 
+class InputEventFilter : public wxEventFilter
+{
+public:
+	InputEventFilter()
+	{
+		wxEvtHandler::AddFilter(this);
+	}
+
+	~InputEventFilter()
+	{
+		wxEvtHandler::RemoveFilter(this);
+	}
+
+	int FilterEvent(wxEvent& event) override;
+
+	void BlockEvents(bool block) { m_block = block; }
+
+private:
+	static bool ShouldCatchEventType(wxEventType type)
+	{
+		// Can't use a switch... "expression must have a constant value"
```

Comment:
This comment suggests you've tried to do something strange, and it didn't work. Doesn't need a comment.